services/QuillLMS/app/assets/stylesheets/shared/footer.scss

```diff
@@ -130,11 +130,20 @@
       }
     }
   }
+  .disclaimer-container {
+    margin: 0 auto;
+    max-width: 1200px;
+    color: #5C5C5C;
```

Comment:
so moving forward, all hex codes should be replaced with the new color variables we have now: 
- https://www.quill.org/backpack
- https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/assets/stylesheets/variables.scss